bin/easystar-0.4.3.js

```diff
@@ -78,9 +78,11 @@ var EasyStar =
 	    var iterationsSoFar;
 	    var instances = {};
 	    var instanceQueue = [];
+	    var instanceQueueIndex = 0;
```

Comment:
Don't change the files in `bin/*`. They will be compiled when the version is bumped.

Reply:
will do. i'll checkout the earlier revision.